app/realtime.php

```diff
@@ -519,6 +519,12 @@ function getCache(): Cache
             ]
         ];
 
+        // Handle SQL error code is 'HY000'
+        $code = $th->getCode();
+        if (!is_int($code)) {
+            $code = 500;
+        }
+
```

Comment:
We should do this before we create the response, and use `$code` instead of `$th->getCode()`

Reply:
Updated!